testutils/package.json

```
@@ -68,7 +68,7 @@
     "@types/jest": "^26.0.10",
     "@types/node-fetch": "^2.5.4",
     "jest-retries": "^1.0.1",
-    "lighthouse": "6.3.0",
+    "lighthouse": "9.2.0",
```

Maybe this could also be switched to `^`?

```suggestion
    "lighthouse": "^9.2.0",
```

---

We don't seem to be running `lighthouse` checks on CI at the moment, and might then be missing issues related to such upgrades.